dashboard/src/app/billing/card-info/card-info.html

```diff
@@ -51,6 +52,7 @@
                che-label-name="State"
                che-name="state"
                che-place-holder=""
+               che-type-city
```

Comment:
It's not validator of cities, it is just directive which prevent to enter not valid symbols to input.